docs/src/components/Sidebar/Sidebar.tsx

```diff
@@ -60,7 +60,9 @@ class Sidebar extends React.Component<any, any> {
     }
 
     return _.findIndex(sections, (section: ShorthandValue<HierarchicalTreeItemProps>) => {
-      return _.find((section as any).items, item => item.title.to.startsWith(newAt))
+      return _.find((section as any).items, item =>
+        new RegExp(`^${newAt}`, 'i').test(item.title.to),
+      )
```

Comment:
I think that we should fix links instead

Reply:
I agree. Let's not create a fix, as @kolaps33 is working on a PR that changes the Accessibility tab. With his changes, we should no longer have a way to click on the broken a11y links.